manifests/base/templates/clusterrole.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: istiod-{{ .Values.global.istioNamespace }}
+  name: istiod-{{ .Release.Namespace }}
```

Comment:
Do we need the suffix at all @dgn @costinm ?

Reply:
Mid-term we'll want the suffix to support multi-tenancy - I'd appreciate having it (we're adding it in Maistra)

Reply:
So each control plane has its own clusterrole then? Because the other stuff, like webhook, already has the revision suffix

Reply:
yes; technically you could be deploying different versions so there can be differences between clusterroles.

now that you're saying it, it might be enough to have the revision I guess

Reply:
Currently the revisions share clusterroles though. We could change it if needed though

Reply:
+1 on supporting multi-tenancy - but by using the istioNamespace to indicate which NS the cluster roles, etc are associated with.